webpack-test/ConfigTestCases.template.js

```diff
@@ -59,7 +59,9 @@ const describeCases = config => {
 			stderr = captureStdio(process.stderr, true);
 		});
 		afterEach(() => {
-			stderr.restore();
+			if (stderr) {
+				stderr.restore();
```

Comment:
Sometimes the test will crash and leave `stderr = undefined` , which crashes jest